docs/tutorials/notebooks/Monge_Gap.ipynb

```diff
@@ -0,0 +1,734 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #98.            batch['target'] = target.generate_samples(rng, self.batch_size)
</pre><p>You shouldn't use the same <code>rng</code> for both, please make sure to split it (would cause issues when both <code>source/target</code> are the same distributions.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/361/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>